woocommerce/woocommerce-ios
# [Order Attribution] Add core data entity to store order attribution info

Closes: #11933

## Description

Adds a new core data entity to store order attribution information. 

## Testing instructions
- CI should pass.
- Install a build from the trunk. Run this branch and replace the trunk build with this build. Ensure that the app works as before without any core data migration crash.

## Screenshots
NA

---
